manifests/s/Splashtop/SplashtopPersonal/3.5.0.3/Splashtop.SplashtopPersonal.locale.en-US.yaml

```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Splashtop.SplashtopPersonal
+PackageVersion: 3.5.0.3
+PackageLocale: en-US
+Publisher: Splashtop Inc.
+PackageName: Splashtop Personal - Remote Desktop for Windows
+License: Copyright (c) Splashtop Inc.

```

Comment:
A copyright isn't a license; Please update this with actual license terms such as Proprietary, MIT, GPL-2.0, or other terms as appropriate

Reply:
yes, we should use Proprietary instead.